### DISCUSSION TITLE
Allow admins to select a default behavior for automatically resolving image upload conflicts

### DISCUSSION THREAD

**Discussion by murtaugh** [ExpressionEngine/ExpressionEngine#2961] [Feature Requests]
Despite education on best practices, clients (and me, if I'm being truthful) often upload images that have a generic name, resulting in the need to resolve name conflicts on a regular basis. What usually happens is one resolution type is used 99.9% of the time, usually "append numeral."

It would be great if, in the settings for the file manager, you could pick a default behavior to automatically resolve conflicts with.

(This would also be great for those times that I'm doing large batches of edits; I could temporarily pick a new default.)

**Comment by intoeetive** [comment]
Will be resolved by https://github.com/ExpressionEngine/ExpressionEngine/pull/3699

Closing as duplicate of https://github.com/ExpressionEngine/ExpressionEngine/discussions/3635
